src/organic.php

```diff
@@ -12,14 +12,34 @@
 
 use Organic\Organic;
 
+// The DSN to use before we load client-specific DSNs.
+const DEFAULT_SENTRY_DSN = 'https://e1cf660e5b3947a4bdf7c516afaaa7d2@o472819.ingest.sentry.io/4505048050434048';
```

Comment:
Sentry (somewhat questionably, I think) says that making the DSN public is ok (see [here](https://docs.sentry.io/product/sentry-basics/dsn-explainer/#dsn-utilization)). I'd prefer not to hard-code this, although I don't know how to do that without having the client copy-and-pasting a DSN, which seems even worse (tedious and error-prone).

Reply:
Looks good! 
It's okay for DSN to be public. In the case of frontend (Platform UI or Organic SDK), there is no other way except hardcoding into a bundle - so, here it's the same 🙂